src/signers/RelaySigner.ts

```diff
@@ -0,0 +1,202 @@
+import { ethers } from 'ethers'
+
+import { TLProvider } from '../providers/TLProvider'
+import { TLSigner } from './TLSigner'
+
+import { Signature, UserObject } from '../typings'
+
+/**
+ * The RelaySigner class contains wallet related methods.
+ */
+export class RelaySigner implements TLSigner {
```

Comment:
Why is this called relay signer, what is relay specific.

Reply:
Hmm yes nothing truly relay specific related here... Maybe rename it to `WalletSigner`, `Wallet` or `TLWallet`?

Reply:
Lets take TL Wallet